### PR TITLE
updates before publish

### PR DESCRIPTION
**Pull request by shefaliv**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@shefaliv__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (shvohra). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@shefaliv version update?

File: src/ResourceManagement/HDInsight/HDInsight/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -31,6 +31,4 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
-// [assembly: AssemblyVersion("0.9.0.0*")]
-[assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.0.2.0")]
```
Comment (yugangw-msft) [inline src/ResourceManagement/HDInsight/HDInsight/Properties/AssemblyInfo.cs]: you should separate out the file version, which will be in different updating cadence with assembly version such as you are just tweaking the build number and want to keep the assembly version so not to break your client app

File: src/ResourceManagement/HDInsightJob/HDInsightJob/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -31,6 +31,4 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
-// [assembly: AssemblyVersion("0.9.0.0*")]
-[assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.0.1.0")]
+[assembly: AssemblyVersion("1.0.1.0")]
```
Comment (yugangw-msft) [inline src/ResourceManagement/HDInsightJob/HDInsightJob/Properties/AssemblyInfo.cs]: same reason

**Pull request merged**